### PR TITLE
Use getfixturevalue instead of getfuncargvalue.

### LINKED ISSUE #97: Avoid getfuncargvalue
getfuncargvalue is deprecated and raises warnings:
https://docs.pytest.org/en/latest/builtin.html#_pytest.fixtures.FixtureRequest.getfuncargvalue

We should stop using it and switch to getfixturevalue instead. getfixturevalue has been around for a while now and I think we can just change it.

### PR DESCRIPTION
**Pull request by pelme**
getfuncargvalue is deprecated and raises warnings.

Fixed #97.

### CODE REVIEW

File: pytest_splinter/plugin.py [hunk 1]
```diff
@@ -501,9 +501,9 @@ def get_browser(splinter_webdriver, retry_count=3):
                 raise
 
     def prepare_browser(request, parent, retry_count=3):
-        splinter_webdriver = request.getfuncargvalue('splinter_webdriver')
-        splinter_session_scoped_browser = request.getfuncargvalue('splinter_session_scoped_browser')
-        splinter_close_browser = request.getfuncargvalue('splinter_close_browser')
+        splinter_webdriver = request.getfixturevalue('splinter_webdriver')
```
Comment (bubenkoff) [inline pytest_splinter/plugin.py]: please update minimum pytest requirements for that this works

Comment (pelme) [reply pytest_splinter/plugin.py]: @bubenkoff is a `pytest`->`pytest>=XXX` change in setup.py enough or would I miss something then?

Comment (pelme) [reply pytest_splinter/plugin.py]: I've specified `pytest>=3.0.0` in setup.py for now.

**Comment by bubenkoff** [comment]
ah, there is a  dependency issue reported by travis, please take a look

**Comment by pelme** [comment]
Hmm: Latest splinter depends on `selenium>=3.4.3`, but requirements-testing.txt has `selenium<3`.

Running with selenium 3 crashes since geckodriver does not exist. I'll open a new issue about this, but it has nothing to do with this PR.

**Comment by pelme** [comment]
I opened #99 to track selenium3/geckodriver compatibility. I don't have the time to fix that right now. :/

**Comment by bubenkoff** [comment]
@pelme thanks, 1.9.0 is out

**Pull request merged**